code/modules/mob/mob_vars.dm

```diff
@@ -250,3 +250,6 @@
 	/// Does this mob speak OOC?
 	/// Controls whether they can say some symbols.
 	var/speaks_ooc = FALSE
+
+	///gunshot residue for det work
```

Comment:
Can you be a bit more descriptive as to what this variable would include?

Reply:
+1

Reply:
Can you be a bit more descriptive as to what this variable would include?